component/autoscaler.jsonnet

```diff
@@ -1,11 +1,61 @@
 local com = import 'lib/commodore.libjsonnet';
+local espejote = import 'lib/espejote.libsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_nodes;
 local metrics = import 'autoscaling-metrics.libsonnet';
 
+local autoscalerArgsPatch = if params.autoscaling.addAutoscalerArgs != null && std.length(params.autoscaling.addAutoscalerArgs) > 0 then
+  assert std.member(inv.applications, 'espejote') : 'The addAutoscalerArgs patch depends on espejote';
+  {
+    autoscaler_inject_args_admission: espejote.admission('autoscaler-inject-args', params.machineApiNamespace) {
+      metadata+: {
+        annotations+: {
+          description: "Injects autoscaler arguments to the default autoscaler pod in the openshift-machine-api namespace.\nArguments are taken from flagsToAdd.\n\nThe patch blindly adds the arguments without trying to replace them if they already exist.\nI debated replacing them but we won't be able to guess all upstream changes in the args array and our args parsing might fail anyways.\nI prefer a simpler patch that fails fast to a convoluted args array merge.\n",
```

Comment:
Could be a multi-line string literal, cf. https://github.com/appuio/component-openshift4-nodes/pull/104/files#diff-122496f6f59ffbaa79899381e538fe6346a434842b963c798b3f0561e41d1c27R108-R135

Reply:
True, pure `yq . -ojson` copy.